contracts/staking/test/unit_tests/exchange_test.ts

```diff
@@ -142,20 +128,6 @@ blockchainTests.resets('Exchange Unit Tests', env => {
             return expect(tx).to.revertWith(expectedError);
         });
 
-        it('should successfully remove an exchange if called by the (authorized) owner', async () => {
```

Comment:
Thanks for fixing these 👍. I think it would make sense to add them back in though, so that we have test coverage to prevent the owner from being as an authorized address in the contracts.